bin/templates/java/org/yarp/Nodes.java.erb

```diff
@@ -18,6 +18,12 @@ public abstract class Nodes {
     }
 
     static final TokenType[] TOKEN_TYPES = TokenType.values();
+<%- flags.each do |flag| -%>
+
+    <%- flag.values.each_with_index do |value, index| -%>
+    static final int <%= flag.human.upcase %>_<%= value.upcase %> = 1 << <%= index %>;
```

Comment:
It should be `public` to be accessible outside of this package.
I'll make a small class for this, based on https://github.com/oracle/truffleruby/blob/master/src/main/java/org/truffleruby/core/regexp/RegexpOptions.java

Reply:
https://github.com/ruby/yarp/pull/948